roles/etcd/templates/etcd-docker.service.j2

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=etcd docker wrapper
-Wants=docker.socket
-After=docker.service
+Wants=docker.service docker.socket
```

Comment:
Wants=docker.service causes docker restart when you systemctl daemon-reload. I made a patch specifiy to avoid this. We should try to prevent gratuitous docker restarts